data/sheragi/the emerald sword.txt

```diff
@@ -0,0 +1,854 @@
+			`	"Call me Miss Deus ex Machina, because I thought ahead and bought these from a merchant back home. Perfect disguise. The humans will never notice. How do I look?"`
+			choice
+				`	"That's... clever. It hides you well."`
+				`	"Well, it works, but you look like a child."`
+					goto child
+			`	"Right? I can even hide my tail in the back of the hoodie!" She spins around happily.`
+				goto anyway
+			label child
+			`	"That's not that bad, is it? I always considered myself young at heart anyway!" she giggles a little.`
+			label anyway
+			`	"Anyway, can you take us to Mr. Foster's museum? We'll take the cube and fill him in on the details."`
+			`	You agree, and after they're all ready, you make your way out to the dig site. Surprisingly, the three Hai don't get many weird looks if they walk a bit further behind you. You arrive at the museum and they immediately break character by being very obviously excited to see the exhibits. This probably comes as a relief to onlookers, however, who would normally expect that type of kid to vandalize priceless artifacts.`
+			`	Finally, you arrive at Albert's workshop. He stops brushing what looks like what was once a television and looks startled at you and the three "kids." The Hai take off their hoods and bandannas, causing his jaw to nearly drop to the floor.`
+			`	"I'll be damned. The Hai really exist!" He exclaims.`
+			`	"Of course we do," Eyama laughs. "Haven't you heard all the stories?"`
+			`	"I had my doubts. Merchants are well known for telling tall tales." He smiles. "Anyway, I'm Albert Foster, archaeologist, pleased to meet you all."`
+			`	They introduce themselves, and very soon the conversation evolves to a kind of jargon even Albert has trouble following. They talk about the box and their findings, while Albert talks about the history uncovered in the dig site and insights on the Sheragi daily life and culture. You close the door to make sure nobody else is startled.`
+			`	"We've managed to open the box and we're making a blueprint of it," says Desyo. "Once that's done, we might be able to rebuild the broken pieces and recover the data. We already have documentation on how their hardware worked."`
+			`	"The catch is that this is a collaboration now. We'll help you, if you share your findings with us," Rakei continues.`
+			`	"Sounds like a win-win to me," Albert says. "We'll arrange the details later. <first>, thank you so much for your help. We'll continue studying this artifact, and we'll send you a message when we're done."`
+			`	You say goodbye to Albert, Rakei, Eyama, and Desyo, and make your way back to your ship. However, as you're walking back to the ship, you feel a hand on your shoulder. You look back, and you're startled to see a Quarg. Yarthis, who you brought here, to be more specific.`
```

Comment:
`Yarthis, who you brought here, to be more specific.`
This is a fragment. Also, `to be more specific` is unnecessary. Just `who you brought here` might not be enought to jog the memory of some players. I'd recommend: `It's Yarthis, who you brought here to help Albert Foster study the Sheragi during the early days of the project.`